arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -326,5 +326,78 @@ await _client.PostVertexCollectionAsync(graphName, new PostVertexCollectionBody(
             Assert.Equal(HttpStatusCode.NotFound, apiError.Code);
             Assert.Equal(1924, apiError.ErrorNum); // ERROR_GRAPH_NOT_FOUND
         }
+
+        [Fact]
+        public async Task PostVertexAsync_ShouldSucceed()
```

Comment:
Let's also add a test to ensure query parameters are working. Example: `PostVertexAsync_ShouldReturnNewVertex_WhenReturnNewIsTrue`